src/main/java/frc/robot/commands/LedToggleCommand.java

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.LedSubsystem;
+
+public class LedToggleCommand extends CommandBase {
```

Comment:
Consider using a [StartEndCommand](https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/wpilibj2/command/StartEndCommand.html#%3Cinit%3E(java.lang.Runnable,java.lang.Runnable,edu.wpi.first.wpilibj2.command.Subsystem...)) instead of this file.